React/game/src/app/components/App/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+import Game from '../Screen/Game';
+
+class App extends Component {
+  render() {
+    return <Game />;
```

Comment:
Same as @DLotero says. Check this https://github.com/Wolox/tech-guides/blob/master/react/style-guide.md#class-vs-reactcreateclass-vs-stateless

Reply:
👍